app/services/gmail/labels.py

```
@@ -168,25 +167,30 @@ def _apply_label_operation_background(
             )
             label_name = label_info.get("name", "")
             if not label_name:
-                state.label_operation_status["done"] = True
-                state.label_operation_status["error"] = "Could not get label name"
+                state.update_label_operation_status(
+                    done=True, error="Could not get label name"
+                )
                 return
         except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)